webapp/channels/src/components/channel_header_mobile/show_search_button/show_search_button.tsx

```diff
@@ -15,24 +15,22 @@ type Props = {
     actions: Actions;
 }
 
-export default class ShowSearchButton extends React.PureComponent<Props> {
-    handleClick = () => {
-        this.props.actions.openRHSSearch();
+export default function ShowSearchButton({actions}: Props) {
```

Comment:
A few minor style things so it is similar to the rest of the files in the app:
- Make this an arrow function
- Do the export at the end of the file
- Since this is a pure component, use React Memo

So... something like...
```
const foo = ({acitons}: Props) => {
    ...
}
export default React.memo(foo);
```

Reply:
@larkox, Thanks for your feedback 🌱.
I have updated the file to match the pattern.